tools/migrators/README.md

```diff
@@ -1,49 +1,55 @@
-# PagerDuty to Grafana OnCall migrator tool
+# Grafana OnCall migrator tools
 
-This tool helps to migrate your PagerDuty configuration to Grafana OnCall.
+These tools will help you to migrate from various on-call tools to Grafana OnCall.
 
-## Overview
+Currently the migration tool supports migrating from:
 
-Resources that can be migrated using this tool:
+- PagerDuty
+- Splunk OnCall (VictorOps)
 
-- User notification rules
-- On-call schedules
-- Escalation policies
-- Services (integrations)
-- Event rules (experimental, only works with global event rulesets)
+## Getting Started
 
-## Limitations
+1. Make sure you have `docker` installed and running
+2. Build the docker image: `docker build -t oncall-migrator .`
```

Comment:
Do we want to start publishing this to save people the build step (separate issue)?

Reply:
Good idea 👍 https://github.com/grafana/oncall/issues/4345